src/App.jsx

```diff
@@ -1,11 +1,53 @@
 import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import DonationForm from './DonationForm';
 
 const App = () => {
-  return (
-    <>
-      <p>Hello World</p>
-    </>
-  );
+  const { backend } = useBackend();
+  var max_id = -1;
+
+  const [formData, setFormData] = useState({
+    business_id: null, // Since there's no business_id in the form and the database requires a business id (foreign key), the default for now is the maximum business id in the database (9)
+    canned_cat_food_quantity: null,
+    canned_dog_food_quantity: null,
+    date: null,
+    donation_id: null,
+    dry_cat_food_quantity: null,
+    dry_dog_food_quantity: null,
+    email: null,
+    food_bank_donation: null,
+    misc_items: null,
+    reporter: null,
+    volunteer_hours: null,
+  });
+
+  const getData = async () => {
```

Comment:
here i see that you are doing this to get the max_id. in the future we will know what id the user is from the authenticationcontext.jsx, so we can use this to get the max_id in the future. for now we can set the MAX_ID as a constant and just set it to 9 for now. you can also put a commment saying to make it not hard coded in the future so its clear why the value is 9. 

sorry to have wasted yalls time on it, but remove this function if possibel since it doesnt look liike you need it??